go.mod

```diff
@@ -32,3 +32,5 @@ require (
 // Fix for `[` in help messages and shell completion code
 // See https://github.com/spf13/cobra/pull/899
 replace github.com/spf13/cobra => github.com/chmouel/cobra v0.0.0-20191021105835-a78788917390
+
+go 1.13
```

Comment:
Is this PR based on #535 ?

Reply:
Yes, will resolve the conflicts at once after 535 merged.

Reply:
Rebased.